app/views/LegalView.tsx

```diff
@@ -9,13 +9,17 @@ import { withTheme } from '../theme';
 import SafeAreaView from '../containers/SafeAreaView';
 import * as List from '../containers/List';
 
-class LegalView extends React.Component {
-	static propTypes = {
-		server: PropTypes.string,
-		theme: PropTypes.string
+interface ILegalView {
+	server: string;
+	theme: string;
+}
+
+class LegalView extends React.Component<ILegalView, any> {
+	static navigationOptions: StackNavigationOptions = {
+		title: I18n.t('Legal')
 	};
 
-	onPressItem = ({ route }) => {
+	onPressItem = ({ route = '' }) => {
```

Comment:
Can you change this part for this: `onPressItem = ({ route }: { route: string }) => {`, please ? thanks :)